### PR TITLE
add hello example

### PR DESCRIPTION
**Pull request by ijt**
This is derived from https://zguide.zeromq.org/docs/chapter1/.

### CODE REVIEW

**Review by sbinet — APPROVED** [review]
LGTM, modulo the small nit-pick.

could you send another PR against [go-zeromq/license](https://github.com/go-zeromq/license) adding yourself to the `AUTHOR` and/or `CONTRIBUTORS` files?

thanks.

File: example/helloclient.go [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+// Hello client
+package main
+
+import (
+	"context"
+	"fmt"
+	zmq "github.com/go-zeromq/zmq4"
+	"log"
+	"time"
+)
```
[the remark this answers is not in the document]
Comment (ijt) [reply example/helloclient.go]: Done.

**Review by sbinet — APPROVED** [review]
LGTM.

(put please send another PR against go-zeromq/license adding yourself to the `AUTHORS` and/or `CONTRIBUTORS` files. then I'll merge this PR in)

**Comment by ijt** [comment]
> these files are in the `license` repo:
> 
> https://github.com/go-zeromq/license

Got that. Pull requests created.

**Comment by sbinet** [comment]
needs https://github.com/go-zeromq/license/pull/10 and https://github.com/go-zeromq/license/pull/11

**Comment by sbinet** [comment]
thanks a lot and welcome to go-zeromq :)

**Pull request merged**